docs-next/src/content/docs/running/cli.mdx

```diff
@@ -0,0 +1,625 @@
+
+Suppose, for example, you have "api" related tests, as well as "app" related tests, as shown in the following snippet; One could use `--grep api` or `--grep app` to run one or the other.
+The same goes for any other part of a suite or test-case title, `--grep users` would be valid as well, or even `--grep GET`.
+
+And another option with double quotes: `--grep "groupA|groupB"`.
+
+And for more complex criteria: `--grep "/get/i"`.
+Some shells as e.g. Git-Bash-for-Windows may require: `--grep "'/get/i'"`.
+Using flags other than the `ignoreCase /i` (especially `/g` and `/y`) may lead to unpredictable results.
+
+```js
+describe("api", function () {
+  describe("GET /api/users groupA", function () {
+    it("respond with an array of users", function () {
+      // ...
+    });
+  });
+});
+
+describe("app", function () {
+  describe("GET /users groupB", function () {
+    it("respond with an array of users", function () {
+      // ...
+    });
+  });
+});
+```
+
+Mutually exclusive with `--fgrep`.
+
+### `--invert`
+
+Use the _inverse_ of the match specified by `--grep` or `fgrep`.
+
+Requires either `--grep` or `--fgrep` (but not both).
+
+## Other Options
+
+### `--inspect, --inspect-brk, inspect`
+
+Enables Node.js' inspector.
+
+Use `--inspect` / `--inspect-brk` to launch the V8 inspector for use with Chrome Dev Tools.
+
+Use `inspect` to launch Node.js' internal debugger.
+
+All of these options are mutually exclusive.
+
+Implies `--timeout 0`.
+
+## About Option Types
+
+:::note[Updated in v6.0.0]
+:::
+
+Each flag annotated of type `[boolean]` in Mocha's `--help` output can be _negated_ by prepending `--no-` to the flag name.
+For example, `--no-color` will disable Mocha's color output, which is enabled by default.
+
+Unless otherwise noted, _all_ boolean flags default to `false`.
+
+## About `node` Flags
+
+The `mocha` executable supports all applicable flags which the `node` executable supports.
+
+These flags vary depending on your version of Node.js.
+
+`node` flags can be defined in Mocha's [configuration](./configuring).
+
+:::note[New in v9.1.0]
+You can also pass `node` flags to Node.js using [`--node-option`](#--node-option-name--n-name).
+:::
+
+### `--enable-source-maps`
+
+:::note[New in vNode.js v12.12.0]
+:::
+
+If the [`--enable-source-maps`](https://nodejs.org/dist/latest-v12.x/docs/api/cli.html#cli_enable_source_maps) flag
+is passed to Mocha, source maps will be collected and used to provide accurate stack traces for transpiled code:
+
+```bash
+Error: cool
+    at Object.<anonymous> (/Users/fake-user/bigco/nodejs-tasks/build/src/index.js:27:7)
+        -> /Users/fake-user/bigco/nodejs-tasks/src/index.ts:24:7
+```
+
+## About V8 Flags
+
+Prepend `--v8-` to any flag listed in the output of `node --v8-options` (excluding `--v8-options` itself) to use it.
+
+V8 flags can be defined in Mocha's [configuration](./configuring).
```

Comment:
I thought this was Mocha v8 at first, was confused, a link might help. Couldn't find a link to an official flags list

```suggestion
[V8](https://v8.dev/) flags can be defined in Mocha's [configuration](./configuring).
```

Reply:
Mark as resolved, just proofreading